app/models/spree/option_value_decorator.rb

```diff
@@ -13,6 +13,6 @@ def has_image?
     image_file_name && !image_file_name.empty?
   end
 
-  default_scope { order("#{quoted_table_name}.position") }
-  scope :for_product, lambda { |product| select("DISTINCT #{table_name}.*").where("spree_option_values_variants.variant_id IN (?)", product.variant_ids).joins(:variants) }
+  default_scope { order(%{#{quoted_table_name}."position"}) }
+  scope :for_product, lambda { |product| select("DISTINCT #{table_name}.*").where(spree_option_value_variants: {variant_id: product.variant_ids}).joins(:variants) }
```

Comment:
Line is too long. [164/80]<br>Use the new lambda literal syntax `->(params) {...}`.<br>Space inside { missing.<br>Space inside } missing.